File: XP.py

nome = "CAboreto"
xp = 5000

if xp < 1000:
    nivel = "Ferro"
elif xp <= 2000:
    nivel = "Bronze"
elif xp <= 5000:
    nivel = "Prata"
elif xp <= 7000:
    nivel = "Ouro"
elif xp <= 8000:
    nivel = "Platina"
elif xp <= 9000:
    nivel = "Ascendente"
elif xp <= 10000:
    nivel = "Imortal"
else:
    nivel = "Radiante"

# Exibe a mensagem final
print(f"O Herói de nome **{nome}** está no nível de **{nivel}**")

File: XP.js

let = "Cabarony"
let xp = 5000 

let level = "";

if (xp < 1000) {
    level = "ferro";
}
else if (xp < 2000) {
    level = "bronze";
}  
else if (xp < 3000) {
    level = "Prata";
}
else if (xp < 4000) {
    level = "Ouro";
}
else if (xp < 5000) {
    level = "platina";
}
else if (xp < 6000) {
    level = "diamante";
}
else if (xp < 7000) {
    level = "Acendente";
}
else if (xp < 8000) {
    level = "Imortal";
}
else {
    level = "Desafiante";
}

console.log(`O jogador ${let} tem ${xp} de XP e está no nível ${level}.`);